### ISSUE TITLE
AUDIT - Menu bar button size

### ISSUE THREAD

**Issue by isabela-pf** [Quansight-Labs/jupyterlab-accessible-themes#19]
## Background

In the menu bar many of the buttons/clickable areas have targets that are too small for WCAG. They need to be bigger.

We need to decide whether we can meet which size in JupyterLab. The larger 44px by 44px is ideal, but it may not work well for many areas of JupyterLab.

From the [3.2.0 review](https://github.com/Quansight-Labs/accessibility/pull/9#issuecomment-1184017454)

> - [ ] The target for clickable areas is at least 24 by 24px. See guidelines for exceptions. [2.5.8](https://www.w3.org/TR/WCAG22/#target-size-minimum) 
     - *Question: is this measured in area? Or is it truly both dimensions need to be at least this?*
     - *Does not meet this: menu bar, file browser breadcrumbs, toc top bar, extension refresh button, tab close, cell collapser, in cell toc collapser, TOC paragraph link*
     - *Does meet this: side bar sections, new launcher (file browser), file browser top bar, file browser search, file browser sort, file browser items, everything in running tab, toc items, table of contents expanded extension button (enable/disable), extension collapser carrot, extension management buttons (about, update, install, etc.), debugger top bars, status bar buttons, tabs, notebook toolbar, one-line cell height, dialog button, dialog drop down*
>  - [ ] The target for clickable areas is at least 44 by 44px. See guidelines for exceptions. [2.5.5](https://www.w3.org/TR/WCAG22/#target-size-enhanced) 
        - *Most places are not meeting this.*
        - *Does meet this: launcher card, two-or-more line cells*

## Tests
- Inspect each clickable area in the browser a5 100% zoom. It needs to be at minimum 24px by 24px.

## References

- [Theme scope checklist](https://github.com/isabela-pf/accessibility/pull/1/files#diff-084cbb6818caeb52700399405bb8c8e8ed366445ad27c7ac83c70aa10f519382)
- [Audit spreadsheet](https://docs.google.com/spreadsheets/d/1yaWY--2zBi7KqPAnMMbJp_aVXFFT8sBDdci8UfpKjO4/edit#gid=0)
